packages/renderer/src/App.svelte

```diff
@@ -53,14 +55,8 @@ router.subscribe(function (navigation) {
   }
 });
 
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore
-window.events?.receive('display-help', () => {
-  router.goto('/help');
-});
-
-window.events?.receive('display-troubleshooting', () => {
-  router.goto('/troubleshooting/repair-connections');
+window.events?.receive('navigate', (navigationRequest: NavigationRequest) => {
+  navigationHandle(navigationRequest.page, navigationRequest.parameters);
```

Comment:
nice to also take care of that

Reply:
I do not get what you want ?

Reply:
just saying that it's nice to have included existing troubleshooting and help navigation into the new system (but as I commented on a line and not the right block you could not understand)